data/sql/updates/pending_db_world/rev_1620115611701757811.sql

```diff
@@ -0,0 +1,59 @@
+-- INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1620115611701757811');
```

Comment:
why commented? we need it

Reply:
Ohh you are right I always comment this out while testing. Fixed!